Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -460,27 +464,22 @@ void doBlink() {
   digitalWrite(BLINK_PIN, blink_state);
 }
 
-#ifdef ENABLE_BUMPER_DEBUG
 void doBumperDebug() {
   // Read the bumper sensors
```

Comment:
There is now a check in setup() for (in this example) publish_bumper_debug_info_freq_hz != 0 where it then advertises the publisher and sets up the callback in the timer.